service/src/test/java/uk/gov/hmcts/reform/fpl/service/email/content/CaseManagementOrderEmailContentProviderTest.java

```diff
@@ -35,6 +39,8 @@
     @Autowired
     private CaseManagementOrderEmailContentProvider caseManagementOrderEmailContentProvider;
 
+    private static final byte[] APPLICATION_BINARY = TestDataHelper.DOCUMENT_CONTENT;
```

Comment:
Do we need this variable? Could we not just directly use `TestDataHelper.DOCUMENT_CONTENT` instead? The name seems slightly more fitting as well for its uses.